File: 05-ciclos/para/para.pas
Program Para;
Var
numero, contador: Integer;

Begin
    Write('Introduzca un numero: ');
    ReadLn(numero);

    For contador := 1 To numero Do
    Begin
        If contador mod 2 = 0 Then
        Begin
            WriteLn(contador, ' es un numero par!');
        End
        Else
        Begin
            WriteLn(contador, ' es un numero impar!');
        End;
    End;
End.

File: 05-ciclos/para/para.py
if __name__ == "__main__":
    numero = int(input("Introduzca un numero: "))

    for contador in range(1, numero):
        if contador % 2 == 0:
            print(str(contador) + " es un numero par!")
        else:
            print(str(contador) + " es un numero impar!")

